.chloggen/mx-psi_enable-component-localhost-fgh.yaml

```diff
@@ -0,0 +1,26 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: otlpreceiver
```

Comment:
This affects more than the otlp receiver right? Any receivers using `ToServer` is affected

Reply:
I'll add an extra API release note on here, and list the components explicitly on the contrib changelog. Does that make sense?

Reply:
Actually, I don't think an API release note is needed here, this does not change anything in confighttp/configrpc, only in the specific components